README.md

```diff
@@ -31,6 +31,33 @@ make build
  ./bin/akoctl collectinfo -n aerospike,olm --path ~/abc/ --cluster-scope=false
 ```
 
+#### Install via krew plugin manager
```

Comment:
Add a section about krew and how to install that.
Just a small description of krew and a link to `krew official doc` and `install krew`